apps/ledger-live-mobile/src/families/cosmos/shared/02-SelectAmount.tsx

```diff
@@ -138,7 +144,7 @@ function DelegationAmount({ navigation, route }: Props) {
                 value={value}
                 onChange={setValue}
                 inputStyle={styles.inputStyle}
-                hasError={error}
+                hasError={isAmountOutOfRange || isNotEnoughBalance}
```

Comment:
The UI should contain as little blockchain logic as possible, these components must keep relying on `error`, and the changes must be done in the `getTransactionStatus` function instead.